flink-connectors/flink-sql-connector-hive-2.3.9/pom.xml

```diff
@@ -48,7 +45,7 @@ under the License.
 		<dependency>
 			<groupId>org.apache.hive</groupId>
 			<artifactId>hive-exec</artifactId>
-			<version>2.3.9</version>
+			<version>3.1.3</version>
```

Comment:
@ren-jq101 Thanks for the PR, but doesn't make much sense to update both connectors to version 3.1.3. Next to that, updating the version numbers is not enough. Since this would effectively drop version 2.3.9 from the codebase, a discussion and vote needs to happen for that on the mailing list. If that vote would pass, there's more then just a version bump in the POM required. It definitely requires changes to the license NOTICE files plus there could be potential code changes required too.